src/weather.h

```diff
@@ -159,6 +101,7 @@ struct weather_sum {
 weather_datum weather_data( weather_type type );
 namespace weather
 {
+void add_weather_datum( weather_datum new_weather );
```

Comment:
```suggestion
void add_datum( weather_datum new_weather );
```
And maybe here as well.